Dockerfile

```diff
@@ -1,35 +1,70 @@
-FROM python:3.9.7-slim@sha256:cd1045dbabff11dab74379e25f7974aa7638bc5ad46755d67d0f1f1783aee101
+# syntax=docker/dockerfile:1.3
 
-ENV PYTHONUNBUFFERED=1 \
-    PYTHONPATH=/app/ \
-    PORT=8888
+FROM python:3.9-slim-bullseye as compile
 
-EXPOSE $PORT
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
```

Comment:
If you put these `RUN`'s into the same command, that'd decrease the layer count for sure, maybe even the runtime/size?

Reply:
https://github.com/mozilla-services/kinto-dist/pull/1937/commits/a01dd77adce9ec85d01498173751693a7dfccef5